playground/backend/cmd/server/controller.go

```diff
@@ -17,12 +17,12 @@ package main
 import (
 	pb "beam.apache.org/playground/backend/internal/api/v1"
 	"beam.apache.org/playground/backend/internal/cache"
+	"beam.apache.org/playground/backend/internal/code_processing"
 	"beam.apache.org/playground/backend/internal/environment"
 	"beam.apache.org/playground/backend/internal/errors"
-	"beam.apache.org/playground/backend/internal/executors"
-	"beam.apache.org/playground/backend/internal/fs_tool"
 	"beam.apache.org/playground/backend/internal/logger"
-	"beam.apache.org/playground/backend/internal/validators"
+	"beam.apache.org/playground/backend/internal/setupers"
```

Comment:
Not sure I like steeper word. Maybe rename it to something like initializer?

Reply:
I thought about `setup_tools`. @KhaninArtur @ilya-kozyrev what do you think about the best name for this package?

Reply:
Let's follow the following structure pattern as we discussed, e.g. `setup_tools/lifecycle/<file.go>`

Reply:
Done.